bears/general/CLOCBear.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises file of or number of files in directory structure
+    with total lines, number comment lines, actual code lines using tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines', 'Number of Empty lines'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('', filename)
+
+    def process_output(self, output, filename, file):
+        lines = output.split('\n')
+        files = (int)(re.search(r'(\d+)', lines[1]).group(1))
+        ignored_files = (int)(re.search(r'(\d+)', lines[2]).group(1))
```

Comment:
Actually this is valid syntax in python. Although, it is not suggested way in modern python with version 3.x. Since `(int)(...)` sounds more like type casting, and `int(...)` sounds more functional, latter is more preferred way.
I've more experience with CXX language, so I mostly use `(int)(...)` syntax.